entries/g/github.com.json

```diff
@@ -12,7 +12,7 @@
     ],
     "documentation": "https://docs.github.com/en/github/authenticating-to-github/securing-your-account-with-two-factor-authentication-2fa/configuring-two-factor-authentication",
     "recovery": "https://docs.github.com/en/github/authenticating-to-github/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials",
-    "notes": "Hardware tokens must have a backup method, either SMS or software token.",
+    "notes": "Hardware tokens must have a backup method, either SMS or software token. By the end of 223 all users who contribute code will need to enable one or more forms of 2fa",
```

Comment:
Apart from 2023 being misspelled, I don't think this is something that fits in a note. The title for notes on 2fa.directory are `Exceptions & Restrictions` which gives an indication on what kind of information the notes are intended to convey. Furthermore people looking for 2FA on GitHub most likely do not need to know that 2FA will be enforced in a year.